### ISSUE TITLE
Feature Request: Badge icon on Linux

### ISSUE THREAD

**Issue by sadalavoaoj** [nylas/N1#408]
Hi there,

Will we get in the future, notifications in icons? 
Example: 
https://lh5.googleusercontent.com/-0WUx2QaNJYw/VdydINbHdiI/AAAAAAAAHmM/Y1YyooKnuG8/w1012-h67-no/elementary%2Bfreya%2Bplank.png

**Comment by louim** [comment]
Actually, there already is, but they are not unified. They are separated by accounts you have connected, which doesn't really make sense.

![capture d ecran 2015-11-16 a 12 07 15](https://cloud.githubusercontent.com/assets/923718/11188784/a071f3c2-8c5a-11e5-9dac-edd46288773e.png)

**Comment by louim** [comment]
Actually I saw you are on Linux in another issue. Please ignore my answer, I'm on mac.

**Comment by bengotow** [comment]
Hi @sadalavoaoj — what Linux distro and window environment are you using? This is most likely blocked on support for this feature in Electron, but I can check.

**Comment by sadalavoaoj** [comment]
Hi @bengotow  i'm running cinnamon 17.2 and i do not see any notification, like @louim  have it.

**Comment by sadalavoaoj** [comment]
Hi there, 

@bengotow how can i unlock that feature? im using cinnamon 17.3 right now.

**Comment by sbenskin** [comment]
Would love to see this implemented, it's probably the last missing feature preventing a move to using this as a primary mail client

**Comment by miguelmich** [comment]
I'm having the same issue on elementary OS 0.3.2 Freya (Ubuntu 14.04)

**Comment by bengotow** [comment]
This will be available in the next N1 release for Unity launcher environments.

**Comment by jackiehluo** [comment]
Hey! We've pushed an update (`0.4.47`) that should fix this issue—let us know if you see it again! Thanks!
